### PR TITLE
Kernel publisher only log info when not connected.  [Fix #1334]

### LINKED ISSUE #1334: Demote kernel error logs to info logs
"Requested unknown event publisher: kernel" and "Cannot connect to kernel. Could not open character device."

### PR DESCRIPTION
**Pull request by mofarrell**

### CODE REVIEW

File: osquery/events/kernel.cpp [hunk 1]
```diff
@@ -40,7 +41,9 @@ void KernelEventPublisher::configure() {
   for (const auto &sub : subscriptions_) {
     if (queue_ != nullptr) {
       auto sc = getSubscriptionContext(sub->context);
-      queue_->subscribe(sc->event_type, sc->udata);
+      if (queue_ != nullptr) {
```
Comment (theopolis) [inline osquery/events/kernel.cpp]: Don't you check for `!= nullptr` on line 42?

Comment (mofarrell) [inline osquery/events/kernel.cpp]: Lol.  Yep  just want to be extra sure.....

**Comment by theopolis** [comment]
Yay!

**Comment by marpaia** [comment]
:dancer: :dancer:

**Pull request merged**